modules/programs/tealdeer.nix

```diff
@@ -4,29 +4,59 @@ with lib;
 let
   cfg = config.programs.tealdeer;
 
-  tomlFormat = pkgs.formats.toml { };
-
   configDir = if pkgs.stdenv.isDarwin then
     "Library/Application Support"
   else
     config.xdg.configHome;
 
+  tomlFormat = pkgs.formats.toml { };
+
+  settingsFormat = let
+    updatesSection = types.submodule {
+      options = {
+        auto_update = mkOption {
+          type = types.bool;
+          default = false;
+          defaultText = literalExpression "false";
+          example = literalExpression "true";
+          description = ''
+            Specifies whether the auto-update feature should be enabled.
+          '';
+        };
+        auto_update_interval_hours = mkOption {
+          type = types.ints.positive;
+          default = 720;
```

Comment:
Same here. 720 is the [default](https://dbrgn.github.io/tealdeer/config_updates.html#auto_update_interval_hours)